<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="8" passed="6">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="23412" started-at="2019-10-19T07:15:10Z" finished-at="2019-10-19T07:15:33Z">
    <groups>
      <group name="regression">
        <method signature="loginTest.TestCase1_verifyTitle()[pri:0, instance:com.vtiger.tests.loginTest@11161c7]" name="TestCase1_verifyTitle" class="com.vtiger.tests.loginTest"/>
        <method signature="loginTest.TestCase2_verifyLogo()[pri:0, instance:com.vtiger.tests.loginTest@11161c7]" name="TestCase2_verifyLogo" class="com.vtiger.tests.loginTest"/>
        <method signature="loginTest.TestCase3_InvalidLogin()[pri:0, instance:com.vtiger.tests.loginTest@11161c7]" name="TestCase3_InvalidLogin" class="com.vtiger.tests.loginTest"/>
        <method signature="loginTest.TestCase4_Login()[pri:0, instance:com.vtiger.tests.loginTest@11161c7]" name="TestCase4_Login" class="com.vtiger.tests.loginTest"/>
        <method signature="loginTest.TestCase1_verifyTitle()[pri:0, instance:com.vtiger.tests.loginTest@9838eb]" name="TestCase1_verifyTitle" class="com.vtiger.tests.loginTest"/>
        <method signature="loginTest.TestCase2_verifyLogo()[pri:0, instance:com.vtiger.tests.loginTest@9838eb]" name="TestCase2_verifyLogo" class="com.vtiger.tests.loginTest"/>
        <method signature="loginTest.TestCase3_InvalidLogin()[pri:0, instance:com.vtiger.tests.loginTest@9838eb]" name="TestCase3_InvalidLogin" class="com.vtiger.tests.loginTest"/>
        <method signature="loginTest.TestCase4_Login()[pri:0, instance:com.vtiger.tests.loginTest@9838eb]" name="TestCase4_Login" class="com.vtiger.tests.loginTest"/>
      </group> <!-- regression -->
      <group name="smoke">
        <method signature="loginTest.TestCase1_verifyTitle()[pri:0, instance:com.vtiger.tests.loginTest@11161c7]" name="TestCase1_verifyTitle" class="com.vtiger.tests.loginTest"/>
        <method signature="loginTest.TestCase4_Login()[pri:0, instance:com.vtiger.tests.loginTest@11161c7]" name="TestCase4_Login" class="com.vtiger.tests.loginTest"/>
        <method signature="loginTest.TestCase1_verifyTitle()[pri:0, instance:com.vtiger.tests.loginTest@9838eb]" name="TestCase1_verifyTitle" class="com.vtiger.tests.loginTest"/>
        <method signature="loginTest.TestCase4_Login()[pri:0, instance:com.vtiger.tests.loginTest@9838eb]" name="TestCase4_Login" class="com.vtiger.tests.loginTest"/>
      </group> <!-- smoke -->
      <group name="sanity">
        <method signature="loginTest.TestCase2_verifyLogo()[pri:0, instance:com.vtiger.tests.loginTest@11161c7]" name="TestCase2_verifyLogo" class="com.vtiger.tests.loginTest"/>
        <method signature="loginTest.TestCase4_Login()[pri:0, instance:com.vtiger.tests.loginTest@11161c7]" name="TestCase4_Login" class="com.vtiger.tests.loginTest"/>
        <method signature="loginTest.TestCase2_verifyLogo()[pri:0, instance:com.vtiger.tests.loginTest@9838eb]" name="TestCase2_verifyLogo" class="com.vtiger.tests.loginTest"/>
        <method signature="loginTest.TestCase4_Login()[pri:0, instance:com.vtiger.tests.loginTest@9838eb]" name="TestCase4_Login" class="com.vtiger.tests.loginTest"/>
      </group> <!-- sanity -->
    </groups>
    <test name="TestonChrome" duration-ms="20506" started-at="2019-10-19T07:15:10Z" finished-at="2019-10-19T07:15:30Z">
      <class name="com.vtiger.tests.loginTest">
        <test-method status="PASS" signature="launchStart(java.lang.String)[pri:0, instance:com.vtiger.tests.loginTest@11161c7]" name="launchStart" is-config="true" duration-ms="5874" started-at="2019-10-19T07:15:10Z" finished-at="2019-10-19T07:15:16Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchStart -->
        <test-method status="FAIL" signature="TestCase1_verifyTitle()[pri:0, instance:com.vtiger.tests.loginTest@11161c7]" name="TestCase1_verifyTitle" duration-ms="36" started-at="2019-10-19T07:15:16Z" finished-at="2019-10-19T07:15:16Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:568)
at org.testng.Assert.assertEquals(Assert.java:578)
at com.vtiger.tests.loginTest.TestCase1_verifyTitle(loginTest.java:37)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestCase1_verifyTitle -->
        <test-method status="PASS" signature="TestCase2_verifyLogo()[pri:0, instance:com.vtiger.tests.loginTest@11161c7]" name="TestCase2_verifyLogo" duration-ms="217" started-at="2019-10-19T07:15:16Z" finished-at="2019-10-19T07:15:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestCase2_verifyLogo -->
        <test-method status="PASS" signature="TestCase3_InvalidLogin()[pri:0, instance:com.vtiger.tests.loginTest@11161c7]" name="TestCase3_InvalidLogin" duration-ms="2527" started-at="2019-10-19T07:15:16Z" finished-at="2019-10-19T07:15:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestCase3_InvalidLogin -->
        <test-method status="PASS" signature="TestCase4_Login()[pri:0, instance:com.vtiger.tests.loginTest@11161c7]" name="TestCase4_Login" duration-ms="11795" started-at="2019-10-19T07:15:19Z" finished-at="2019-10-19T07:15:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestCase4_Login -->
        <test-method status="PASS" signature="tierdown()[pri:0, instance:com.vtiger.tests.loginTest@11161c7]" name="tierdown" is-config="true" duration-ms="1" started-at="2019-10-19T07:15:30Z" finished-at="2019-10-19T07:15:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tierdown -->
      </class> <!-- com.vtiger.tests.loginTest -->
    </test> <!-- TestonChrome -->
    <test name="TestonFirefox" duration-ms="23412" started-at="2019-10-19T07:15:10Z" finished-at="2019-10-19T07:15:33Z">
      <class name="com.vtiger.tests.loginTest">
        <test-method status="PASS" signature="launchStart(java.lang.String)[pri:0, instance:com.vtiger.tests.loginTest@9838eb]" name="launchStart" is-config="true" duration-ms="11678" started-at="2019-10-19T07:15:10Z" finished-at="2019-10-19T07:15:22Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchStart -->
        <test-method status="FAIL" signature="TestCase1_verifyTitle()[pri:0, instance:com.vtiger.tests.loginTest@9838eb]" name="TestCase1_verifyTitle" duration-ms="19" started-at="2019-10-19T07:15:22Z" finished-at="2019-10-19T07:15:22Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:568)
at org.testng.Assert.assertEquals(Assert.java:578)
at com.vtiger.tests.loginTest.TestCase1_verifyTitle(loginTest.java:37)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestCase1_verifyTitle -->
        <test-method status="PASS" signature="TestCase2_verifyLogo()[pri:0, instance:com.vtiger.tests.loginTest@9838eb]" name="TestCase2_verifyLogo" duration-ms="230" started-at="2019-10-19T07:15:22Z" finished-at="2019-10-19T07:15:22Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestCase2_verifyLogo -->
        <test-method status="PASS" signature="TestCase3_InvalidLogin()[pri:0, instance:com.vtiger.tests.loginTest@9838eb]" name="TestCase3_InvalidLogin" duration-ms="2366" started-at="2019-10-19T07:15:22Z" finished-at="2019-10-19T07:15:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestCase3_InvalidLogin -->
        <test-method status="PASS" signature="TestCase4_Login()[pri:0, instance:com.vtiger.tests.loginTest@9838eb]" name="TestCase4_Login" duration-ms="9082" started-at="2019-10-19T07:15:24Z" finished-at="2019-10-19T07:15:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestCase4_Login -->
        <test-method status="PASS" signature="tierdown()[pri:0, instance:com.vtiger.tests.loginTest@9838eb]" name="tierdown" is-config="true" duration-ms="1" started-at="2019-10-19T07:15:33Z" finished-at="2019-10-19T07:15:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tierdown -->
      </class> <!-- com.vtiger.tests.loginTest -->
    </test> <!-- TestonFirefox -->
  </suite> <!-- Suite -->
</testng-results>
